### PR TITLE
Add project info to README

### PR DESCRIPTION
**Pull request by gliargovas**
Added project info, installation instructions, logo and examples.

### CODE REVIEW

**Review by angelhof — APPROVED** [review]
Looks good to me! I think that the current logo does not have a lot of info bits, but it should be OK for now.

**Comment by mgree** [comment]
Logo seems fine! If we want slickness, we should buckle up and pay a professional designer (maybe to make something in line with the existing pash logo).

For installation instructions, shouldn't it just be "copy the executable into your `$PATH`"?

File: README.md [hunk 1]
```diff
@@ -1,8 +1,96 @@
...
+$ sudo cp try/try /usr/local/bin
+```
+
 ## Example Usage
 
+`try` is a higher-order command, like `xargs`, `exec`, `nohup`, or `find`. For example, to ungzip file, you can invoke `try` as follows:
+
+```ShellSession
+$ try gunzip file.txt.gz
+...
```
Comment (mgree) [inline README.md:52]: TODO: show the output

File: README.md [hunk 2]
```diff
@@ -1,8 +1,96 @@
...
+/tmp/tmp.0caZdxnHuR/upperdir/home/me/file.txt.gz
+
+Commit these changes? [y/N] y
+```
+
+Sometimes, you might want to pre-execute a command and commit its result at a later time. Invoking *try* with the -n flag will return the overlay directory, wothout committing the result.
+
+```ShellSession
+$ try -n gunzip file.txt.gz
+```
```
Comment (mgree) [inline README.md:71]: show output

File: README.md [hunk 3]
```diff
@@ -1,8 +1,96 @@
...
+
+Alternatively, you can specify your own overlay directory as follows (note that *try_dir* already exists):
+
+```ShellSession
+$ try -D try_dir gunzip file.txt.gz
+```
```
Comment (mgree) [inline README.md:77]: use a more realistic `try_dir`, and show what's in it?

File: README.md [hunk 4]
```diff
@@ -1,8 +1,96 @@
...
+
+You can inspect the changhes made inside a given overlay directory:
+
+```
+try summary try_dir
+```
```
Comment (mgree) [inline README.md:83]: turn into a `ShellSession` code block; show the output

File: README.md [hunk 5]
```diff
@@ -1,8 +1,96 @@
...
+
+You can also choose to commit the overlay directory contents:
+
+```
+try commit try_dir
```
Comment (mgree) [inline README.md:88]: `ShellSession`, code block

File: README.md [hunk 6]
```diff
@@ -1,8 +1,96 @@
 # try
 
+<img src="docs/try_logo.png" alt="try logo" width="100" height="130">
+
 "Do, or do not. There is no try."
 
 We're setting out to change that.
```
Comment (mgree) [inline README.md:8]: Can we generate a screen cap gif that demos the basic functionality like I have for `ffs`? Happy to brainstorm on what exactly goes in it.

**Review by mgree — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (gliargovas) [reply README.md:52]: The command output continues on the next code block. Do we also want it here?

Comment (mgree) [reply README.md:52]: That's why I put the `...` there. You could change it to `... # output continued below`.

File: README.md (hunk 6 above)
Comment (gliargovas) [reply README.md:8]: Added an example using pip install 🚀

**Comment by angelhof** [comment]
Merging to work together on this

**Pull request merged**